docker/test/integration/runner/Dockerfile

```diff
@@ -47,6 +47,8 @@ ENV TZ=Etc/UTC
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
 ENV DOCKER_CHANNEL stable
+# Unpin the docker version after the release 24.0.3 is released
```

Comment:
I see only a comment, but where is the pin?

Reply:
oh my, you are right